workshops/login_auth/README.md

```diff
@@ -0,0 +1,501 @@
+---
+name: 'Web Login Authentication'
+description: 'Build a secure web login page using FIREBASE'
+author: '@tanishq-soni'
+---
+
+# Web Login Authentication 🔒
+
+In this workshop you'll build a secure web login page using [FIREBASE](https://firebase.google.com/).
+
+You will make something like this 👇
+
+![live gif](https://cloud-7782l8k04.vercel.app/loginv2.gif)
+
+[Live Example](https://loginauth-12.web.app)
+
+Try live example using:
+- EMAIL-`login@authentication.com`
+- PASSWORD-`password`
+
+## Getting Started :rocket:
+
+
+### Creating a FIREBASE project 💻 -
+At first you will need to sign-in to [FIREBASE](https://firebase.google.com/), if you have google account then you can directly sign-in to FIREBASE otherwise create one.
```

Comment:
```suggestion
At first you will need to sign in to [Firebase](https://firebase.google.com/), if you have Google account then you can directly sign-in to Firebase otherwise create one.
```